docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,167 @@
+# Experimental Code
+
+The team occasionally will author code, or accept contributions, that is
```

Comment:
```suggestion
The team will occasionally author code or accept contributions, that are considered experimental or unstable. The goal for this code is to ship it as unstable for sponsor products to leverage. During this time, the team can get feedback on what is working and what does not work so that changes can be made before an official release.
```

Grammarly check.

Reply:
sponsor products are what please? :)

Reply:
The product/s that we are working for. Lastly, there were Jobs. Currently, it will be the new platform, that we will sponsor. Or pick up/suggest a better copy, how to say that we are supporting some product for a limited amount of time.